pkg/inject/jaeger.go

```diff
@@ -13,13 +13,14 @@ tracing:
    "@type": type.googleapis.com/envoy.config.trace.v2.ZipkinConfig
    collector_cluster: jaeger
    collector_endpoint: "/api/v1/spans"
+   collector_endpoint_version: HTTP_JSON
    shared_span_context: false
 static_resources:
   clusters:
   - name: jaeger
     connect_timeout: 1s
-    type: strict_dns
```

Comment:
Looks good. You will probably need the same change for datadog tracing config[1]

[1] https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/pkg/inject/datadog.go#L21